metrics/metrics.go

```diff
@@ -195,6 +195,8 @@ func RegisterMetrics() {
 	prometheus.MustRegister(AdvancerOwner)
 	prometheus.MustRegister(AdvancerTickDuration)
 	prometheus.MustRegister(GetCheckpointBatchSize)
+	prometheus.MustRegister(RegionCheckpointRequest)
```

Comment:
shall we add metrics json file in this PR?

Reply:
Those metrics are added into the TiKV panel...

Reply:
ok